### PR TITLE
Added-views

### PR DESCRIPTION
**Pull request by moise10r**
### In this PR I did the following things:
- Implement the design from the sneak peek [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).
- Added the style.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = %w[moise rushanika Nganulo]
```
Comment (arslanbisharat) [inline app/controllers/users_controller.rb]: - [ ] Let's update the method from    `@users = %w[moise rushanika Nganulo] `to   `@users = User.all`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+  end
 
-  def show; end
+  def show
+    @author_id = params[:id]
```
Comment (arslanbisharat) [inline app/controllers/users_controller.rb]: - [ ] Let's not use @author_id. It would be good if you can use @user.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @moise10r, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**